index.bs

```diff
@@ -5787,6 +5787,18 @@ The attestation certificate MUST have the following fields/extensions:
     are both OPTIONAL as the status of many attestation certificates is available through authenticator metadata services.
     See, for example, the FIDO Metadata Service [[FIDOMetadataService]].
 
+### Certificate Requirements for Enterprise Packed Attestation Statements ### {#sctn-enterprise-packed-attestation-cert-requirements}
+
+There are two potential sources for Enterprise Attestestations in Packed format:
+
+1. From a hardware device which is manufactured with a device-specific attestation statement and key
+2. Via a provisioned attestation statement, such as with a platform authenticator configured via managed policy
+
+For attestation statements provisioned at manufacturing, the Extension OID `1.3.6.1.4.1.45724.1.1.2` (`id-fido-gen-ce-sernum`) 
```

Comment:
You enumerate two cases just above, but then only mention one here. Are provisioned enterprise attestations forbidden from using this extension? Or can either use them?

Reply:
I'd raise to someone more familiar such as @ve7jtb

Reply:
We are still pending guidance here on usage for MDM-provisioned EAs. Since these can be dynamic (reprovisioned by the MDM or by switching which software provides the credentials), I would lean toward either saying this should not be used, or that it must indicate a persistent device identifier (and not some software value).

Reply:
will update to recommend against this for software-provisioned EAs